### PR TITLE
Add test case for state.go

### PR DESCRIPTION
**Pull request by changshuchao**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/cm/cpumanager/state/state_test.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+func TestClone(t *testing.T) {
+	expect := ContainerCPUAssignments{
+		"pod": map[string]cpuset.CPUSet{
+			"container1": cpuset.NewCPUSet(4, 5, 6),
+			"container2": cpuset.NewCPUSet(1, 2, 3),
+		},
+	}
+	actual := expect.Clone()
+	assert.Equal(t, expect, actual)
```
Comment (wzshiming) [inline pkg/kubelet/cm/cpumanager/state/state_test.go:33]: How to ensure that `expect` and `actual ` are not the same pointer?

Comment (wzshiming) [reply pkg/kubelet/cm/cpumanager/state/state_test.go:33]: Mayby like this 
``` go
if expect == actual || reflect.DeepEqual(expect, actual) {
   t.Fail()
}
```

File: pkg/kubelet/cm/cpumanager/state/state_test.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```
Comment (lala123912) [inline pkg/kubelet/cm/cpumanager/state/state_test.go:2]: 2018 ->2021

File: pkg/kubelet/cm/cpumanager/state/state_test.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package state
+
+import (
+	"github.com/stretchr/testify/assert"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
+	"testing"
```
Comment (lala123912) [inline pkg/kubelet/cm/cpumanager/state/state_test.go:22]: suggested order :
```	
        "testing"

        "github.com/stretchr/testify/assert"

	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
```

**Review by ehashman — COMMENTED** [review]
/ok-to-test
/triage accepted
/priority backlog

**Comment by changshuchao** [comment]
@ehashman hi, i need lgtm and approval label :)

**Comment by changshuchao** [comment]
@klueska hi，could you please review this pr? thanks

**Comment by changshuchao** [comment]
@klueska  can this pr be merged？

**Comment by wzshiming** [comment]
@changshuchao Code freeze during this time, PRs that are not important will not be merged.

**Comment by changshuchao** [comment]
> 
> 
> @changshuchao Code freeze during this time, PRs that are not important will not be merged.

got it

**Pull request merged**